stat_vars/rff_weather_variability.mcf

```diff
@@ -0,0 +1,82 @@
+Node: dcid:StandardDeviation_DailyPrecipitation
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Place
+measuredProperty: dcs:precipitation
+statType: dcs:StandardDeviation
```

Comment:
`standardDeviation`

statType refers to the property you defined in dcschema.mcf and entities are case-sensitive.

by schema.org convention, properties start with lower-case while classes/types start with upper-case.